azure.yaml

```diff
@@ -13,7 +13,19 @@ description: |
 
   ## What you'll need to connect
 
-  **Authentication**: OAuth with web browser authentication - no manual configuration required.
+  **Authentication:**
+  You will need to set the following environment variables:
+  - AZURE_TENANT_ID
+  - AZURE_CLIENT_ID
+  - AZURE_CLIENT_SECRET
+  These must correspond to a service principal. You can create one using the Azure web console, or with the Azure CLI:
+
+  ```bash
+  az ad sp create-for-rbac --name "my-service-principal" --role contributor \
+  --scopes /subscriptions/your-subscription-id \
```

Comment:
minor nit, I think it becomes more obvious is you put things you need to substitute in `<>`.

Reply:
made this change